.github/workflows/nightly_e2e.yml

```diff
@@ -8,6 +8,7 @@ on:
 jobs:
   e2e:
     runs-on: ubuntu-latest
+    if: ${{ secrets.METAL_AUTH_TOKEN != '' && secrets.EQUINIX_ORG_ID != '' }}
```

Comment:
Tried to add a similar check on the docs publish job, but secrets can't be used in condition.

https://github.com/actions/runner/issues/520

Reply:
This logic is a valid workaround:

```
    if: ${{ github.event_name != 'pull_request' && github.repository_owner == 'weaveworks' }}
```

Reply:
boooo stack overflow lied to me 😭 

okay ty